code/modules/projectiles/projectile/bullets.dm

```diff
@@ -124,6 +124,21 @@
 	damage = 10
 	weaken = 4
 	stun = 4
+	
+/obj/item/projectile/bullet/honker
+	damage = 0
+	weaken = 5
+	stun = 5
+	forcedodge = 1

```

Comment:
Precisely. Bikehorn is only a local sound and theres not exactly a ton of mobs on the station, youll get 2-3 honks per shot if you're lucky.